### PR TITLE
Updated Meta

### PR DESCRIPTION
**Pull request by Fortifier42**
Removed double oxygen tags

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 1]
```diff
@@ -521,10 +532,10 @@ public void registerCoreMembers() {
         // Use to temporarily load a chunk.
         // - chunkload ch@0,0,world duration:5m
         // @Usage
-        // Use to stop loading a chunk.
+        // Use to remove a loaded chunk.
         // - chunkload remove ch@0,0,world
         // @Usage
-        // Use to stop loading any chunks.
+        // Use to remove all manually loaded chunks.
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: Neither wordings are ... sane at all... for any of the above usages.

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 2]
```diff
@@ -922,15 +934,20 @@ public void registerCoreMembers() {
         // @Syntax execute [as_player/as_op/as_npc/as_server] [<Bukkit command>] (silent)
         // @Required 2
         // @Stable stable
-        // @Short Executes an arbitrary server command as if the player, NPC, or server typed it in.
+        // @Short Executes an arbitrary server command as if the player, NPC, an op or the server typed it in.
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: Was better in original form.

File: src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 3]
```diff
@@ -2312,31 +2372,24 @@ public void registerCoreMembers() {
...
         // - schematic rotate name:MySchematic angle:90
-
         // @Usage
         // Use to paste a loaded schematic
         // - schematic paste name:MySchematic <player.location> noair
-
         // @Usage
         // Use to save a created schematic
         // - schematic save name:MySchematic
-
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: @ All the above deletes: STAHP. Those newlines are there intentionally!

File: src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java [hunk 4]
```diff
@@ -118,6 +118,17 @@ public void serverTag(ReplaceableTagEvent event) {
...
+        // @returns Element(boolean)
+        // @description
+        // returns true if the server's whitelist is active, otherwise returns false.
+        // -->
+        if (attribute.startsWith("has_whitelist")) {
+            event.setReplaced(new Element(Bukkit.hasWhitelist()).getAttribute(attribute.fulfill(1)));
+            return;
+        }
+
+        // <--[tag]
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: Didn't... we already pull... what?

Comment (Fortifier42) [inline src/main/java/net/aufdemrand/denizen/tags/core/ServerTags.java]: yeah I didn't understand either

**Pull request closed without merging**